network

```diff
@@ -97,7 +97,7 @@ function blockscout-help () {
     echo -e "  ${B}blockscout pull${R}   pull container images for all blockscout services"
     echo -e "  ${B}blockscout start${R}  start block scope instances for the EVM chains"
     echo -e "  ${B}blockscout stop${R}   stop block scope instances for the EVM chains"
-    echo -e "  (The UIs are served at http://localhost:8000 and http://localhost:8001.)"
+    echo -e "  (The UIs are served at http://chain-20.evm.kadena.local:8000 and  http://chain-21.evm.kadena.local:8000 and so on)"
```

Comment:
I think it is still a good idea to mention this in the blockscout/README.md right after the section about the add-domains script. Its easy to miss this in the terminal if you run something else right after it. Right now in blockscout/README.md, the only actual link to the Blockscout explorer opens up http://localhost:8000/, which fails if you have just run the add-domains script.